<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ФИО</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Название проекта</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Задание </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Дата выполнения </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Даты начала</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="output">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>341</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>121</width>
      <height>205</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="inputs">
     <item row="6" column="0">
      <widget class="QLineEdit" name="editEndDate"/>
     </item>
     <item row="2" column="0">
      <widget class="QLineEdit" name="editProjectName"/>
     </item>
     <item row="0" column="0">
      <widget class="QLineEdit" name="editEmployeeName"/>
     </item>
     <item row="4" column="0">
      <widget class="QLineEdit" name="editDateOfCompletion"/>
     </item>
     <item row="3" column="0">
      <widget class="QLineEdit" name="editTask"/>
     </item>
     <item row="5" column="0">
      <widget class="QLineEdit" name="editStartDate"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="openButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>170</y>
      <width>341</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Открыть</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveFileButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>200</y>
      <width>341</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addItemButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>230</y>
      <width>341</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>260</y>
      <width>341</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Количество дней работы</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sortButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>290</y>
      <width>341</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Сортировка </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Даты окончания</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sortButton_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>320</y>
      <width>341</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Выйти</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
